lib/serverengine/process_manager.rb

```diff
@@ -211,7 +214,11 @@ def tick(blocking_timeout=0)
       if ready_pipes
         ready_pipes.each do |r|
           begin
-            r.read_nonblock(1024, @read_buffer)
+            if $platformwin
+              r.read(1024, @read_buffer)
```

Comment:
I changed to use read with Overlapped IO.
This is almost same as readpartial.
But if using readpartial, it blocks when there is nothing to read.
Then we can't use readpartial because there may not be anything from heartbeat client.

So I think that I had better use this Overlapped IO in this situation.
(we can set overlapped flag in open mode for named pipe using FILE_FLAG_OVERLAPPED like this.
https://github.com/naritta/serverengine/blob/windows-signal-accept/lib/serverengine/process_manager.rb#L158)